File: fe_public/quadra_mall_web-main/src/types/home/product.ts
export interface ProductCardDTO {
  id: number;
  name: string;
  slug: string;
  price: number;
  thumbnailUrl: string;
  rating: number;
  soldCount: number;
  isFav: boolean;
  seller: {
    id: number;
    name: string;
    province: string;
    slug: string;
  };
}

File: mobile_app/quadra_mall_app-main/lib/models/product.dart
class Product {
  final String name;
  final double price;
  final String imageUrl;
  final double rating;
  final int soldCount;
  final String seller;
  bool isFav;

  Product({
    required this.name,
    required this.price,
    required this.imageUrl,
    required this.rating,
    required this.soldCount,
    required this.seller,
    this.isFav = false,
  });
}